__tests__/app.test.js

```diff
@@ -202,3 +200,27 @@ describe('GET /api/users', () => {
   });
 })
 
+describe('GET /api/articles', () => {
+  test('sends GET to articles endpoint -> responds with array', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toBeInstanceOf(Array);
+      });
+  });
+
+  test('sends GET to users endpoint -> responds with object with two properties', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((res) => {
+        const body = res.body;
+        body.forEach(article => {
+          expect(article).toBeInstanceOf(Object);
+          expect(Object.keys(article).length).toBe(7);
```

Comment:
Either way, I'd rather you were testing for what types those properties refer to, rather than how many there are. If we use some sort of syntax that allows us to check if the whole article matches a certain format (including types), then there's no need to also test for how many properties are in there.